core/operation/src/main/java/uk/gov/gchq/gaffer/operation/impl/Limit.java

```diff
@@ -32,7 +32,7 @@
 /**
  * A {@code Limit} operation takes in an {@link Iterable} of items
  * and limits the iterable to a given number of items. It simply wraps the input
- * iterable in a {@link uk.gov.gchq.gaffer.commonutil.iterable.LimitedIterable} so
+ * iterable in a LimitedIterable so
```

Comment:
Shouldn't this be a link to `uk.gov.gchq.koryphe.iterable.LimitedIterable`, rather than no link at all?

Reply:
I think check style was moaning at me so I removed. I'll add it bacon and see what's up.